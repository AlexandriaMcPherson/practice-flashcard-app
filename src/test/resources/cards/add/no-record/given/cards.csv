id, card_front, card_back, notes, time_due, review_interval, correct_in_a_row, ease
