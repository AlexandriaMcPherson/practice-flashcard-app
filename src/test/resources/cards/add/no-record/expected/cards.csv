id, card_front, card_back, notes, time_due, review_interval, correct_in_a_row, ease
2, add, 追加,note to self,2023-10-16 06:25:47,1.0,0,2.5